packages/cli/src/lib/cmds/help.ts

```diff
@@ -101,6 +103,16 @@ const HELP_MESSAGES = new Map<string, string>(
         '',
         'bubblewrap doctor',
       ].join('\n')],
+      ['merge', [
+        'Usage:',
+        '',
+        '',
+        'bubblewrap merge',
+        '',
+        '',
+        'Options: ',
+        '--ignore [fields-list]................. the fields which you would like to keep the same',
```

Comment:
Where can I find the fields I can add to `fields-list`?

Reply:
Added explanation on which fields are valid (all of the fields in the web manifest)

Reply:
Here's a list of fields available in the Web Manifest: https://www.w3.org/TR/appmanifest/#webappmanifest-dictionary - It seems the ones we care about are:
 - `name`,
 - `short_name`,
 - `icons`,
 - `start_url`,
 - `display`,
 - `orientation`,
 - `theme_color`,
 - `background_color`,
 - and `shortcuts`.

The following fields are currently not used by Bubblewrap at all :
 - `dir`,
 - `lang`,
 - `description`,
 - `screenshots`,
 - `categories`,
 - ` iarc_rating_id`,
 - `scope`,
 - `related_applications`,
 -  and `prefer_related_applications` 

It's probably OK to support the whole list, but may be useful to point out which ones are used by Bubblewrap (on a following PR)